lib/board_selection.py

```diff
@@ -0,0 +1,18 @@
+from .tictactoe_board import TicTacToeBoard
+
+
+class BoardSelection:
+
+    valid_board_sizes = [3, 4]
+
+    def __init__(self, ui):
+        self.ui = ui
+
+    def select_board_size(self):
+        self.ui.display("Welcome to TIC TAC TOE!")
+        self.ui.display("First, select a board size.")
+        size = int(self.ui.get_user_input())
+        while size not in self.valid_board_sizes:
+            self.ui.display("The size of the board should be 3 or 4, try again!")
```

Comment:
Last bit of feedback - how can we make this message dependent on the values in `valid_board_sizes`? If we add to `valid_board_sizes` then as it stands, we'd have to modify this message as well.

Reply:
Done!